source/fundamentals/connection.txt

```diff
@@ -279,18 +279,29 @@ using a method in the ``MongoClientSettings.Builder`` class.
 Compression Algorithm Dependencies
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+.. note::
+
+   Applications that require Snappy or Zstandard compression must
+   :ref:`add explicit dependencies <compression-dependencies>` for those
+   algorithms.
```

Comment:
[optional suggestion] I thought it was a little confusing to be linked to the section I'm in from this note - if I'm understanding correctly, it looks like it applies to the text just below. Does it make sense to add this sentence, without the link, to that paragraph?

"The JDK supports Zlib compression natively, but Snappy and Zstandard depend upon open source implementations. **Applications that require Snappy or Zstandard compression must add explicit dependencies for those algorithms.** See snappy-java and zstd-java for details."